aws-ssm-maintenancewindow/pom.xml

```diff
@@ -28,7 +28,7 @@
         <dependency>
             <groupId>software.amazon.cloudformation</groupId>
             <artifactId>aws-cloudformation-rpdk-java-plugin</artifactId>
-            <version>[2.0.0, 3.0.0)</version>
+            <version>2.0.16</version>
```

Comment:
Will this upgrade in version will have any effect on any of the existing functionality. Asking as it seems its getting updated for all SSM service here.

Reply:
good point. I think [2.0.0, 2.0.16] might be a better update, since 2.1.0 was the version causing issues.